github/orgs_teams.go

```diff
@@ -44,19 +44,46 @@ func (t Team) String() string {
 	return Stringify(t)
 }
 
-// Invitation represents a team member's inviation status
+// Invitation represents a team member's invitation status.
 type Invitation struct {
-	ID        *int       `json:"id,omitempty"`
-	Login     *string    `json:"login,omitempty"`
-	Email     *string    `json:"email,omitempty"`
+	ID    *int    `json:"id,omitempty"`
+	Login *string `json:"login,omitempty"`
+	Email *string `json:"email,omitempty"`
+	// Role can be one of the values - direct_member, admin, billing_manager, hiring_manager, or reinstate.
 	Role      *string    `json:"role,omitempty"`
 	CreatedAt *time.Time `json:"created_at,omitempty"`
+	Inviter   *Inviter   `json:"inviter,omitempty"`
```

Comment:
Use `User` instead of `Inviter` here:

```Go
Inviter   *User   `json:"inviter,omitempty"`
```

See here for precedent:

https://github.com/google/go-github/blob/c9c37fd6/github/repos_invitations.go#L15